.dotnet/tests/Samples/Chat/Sample05_ChatWithVision.cs

```diff
@@ -2,24 +2,33 @@
 using OpenAI.Chat;
 using System;
 using System.Collections.Generic;
+using System.IO;
 
 namespace OpenAI.Samples
 {
     public partial class ChatSamples
     {
         [Test]
         [Ignore("Compilation validation only")]
-        public void Sample05_ChatWithVision(Uri imageUri = null)
+        public void Sample05_ChatWithVision()
         {
             ChatClient client = new("gpt-4-vision-preview", Environment.GetEnvironmentVariable("OPENAI_API_KEY"));
 
-            List<ChatRequestMessage> messages = [
-                new ChatRequestUserMessage(
-                    "Describe this image for me",
-                    ChatMessageContent.FromImage(imageUri))
+            string imageFilename = "variation_sample_image.png";
+            string imagePath = Path.Combine("Assets", imageFilename);
+            using Stream imageStream = File.OpenRead(imagePath);
+            BinaryData image = BinaryData.FromStream(imageStream);
+
+            List<ChatMessage> messages = [
+                new UserChatMessage(
+                    ChatMessageContentPart.CreateTextMessageContentPart("Please describe the following image."),
```

Comment:
given these methods are static on the type, is it viable for us to remove the duplicative naming segments? `ChatMessageContentPart.CreateText` seems clear enough to me for a static factory.

Reply:
I'll add this comment to the APIView.